ERROR | 2022-07-24 12:16:44,190 | main.py:41 | BOT WAS STARTED
NoneType: None
